schemas/questionnaire_v1.json

```diff
@@ -72,9 +72,9 @@
           "validator": {
             "type": "string",
             "enum": [
+              "boolean",
               "date",
-              "string",
-              "object"
+              "string"
```

Comment:
While you are in this code we have this validator defined that isn't allow in through here
https://github.com/ONSdigital/eq-survey-runner/pull/1715/files#diff-f8f1a812b2bb69e21c3e107b2626e63eR39

Reply:
Sorry, are you suggesting to allow it through, or remove the UUID type from runner?

Reply:
Allow I guess. Either that or remove the code? 
Maybe remove as it’s obviously not used?
Are any of the items currently marked as string actually uuid, if so do we actually care if they aren’t uuid?

Reply:
The only thing documented as being a UUID is `case_id`, however that's not actually validated because it's optional and just passed through as a string. 

I did actually write unit tests for the UUID validation yesterday because I needed to write unit tests for the new boolean validator, so we can be confident that the validation works. We may as well leave it in given that it's a general purpose product?